test/e2e/es/mutation_test.go

```diff
@@ -292,9 +293,24 @@ func TestMutationWhileLoadTesting(t *testing.T) {
 			metrics.Close()
 			bytes, _ := json.Marshal(metrics)
 			msgAndArgs := []interface{}{"metrics: ", string(bytes)}
-			assert.Equal(t, 1, len(metrics.StatusCodes), msgAndArgs)
-			if _, ok := metrics.StatusCodes["401"]; !ok {
-				assert.Fail(t, "all status codes should be 401", msgAndArgs)
+			switch len(metrics.StatusCodes) {
+			case 1:
+				if _, ok := metrics.StatusCodes["401"]; !ok {
+					assert.Fail(t, "all status codes should be 401", msgAndArgs)
+				}
+			case 2:
+				// allow 5 or less network errors during load testing, as we randomly see these during testing.
```

Comment:
Let's maybe create an issue to investigate the reason for these failures and link to it here, so that we don't accept this behaviour as given.